WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -170,6 +213,29 @@ final class ProductDetailPreviewViewModel: ObservableObject {
         shouldShowFeedbackView = false
     }
 
+    // MARK: Switch options
+    func next() {
+        guard let generatedAIProduct else {
+            return
+        }
+
+        guard selectedOptionIndex < generatedAIProduct.names.count - 1 else {
+            return
+        }
+
+        saveCurrentOption()
+        selectedOptionIndex = selectedOptionIndex + 1
+    }
+
+    func previous() {
```

Comment:
Similar comment as above:

```suggestion
    func switchToPreviousOption() {
```

Reply:
Done in 163c44fb3ac21a2db3147551b850e0d4de2221a7